ks3-standard/src/commonMain/kotlin/io/ks3/standard/base64/ByteArrayAsBase64StringSerializer.kt

```diff
@@ -0,0 +1,23 @@
+package io.ks3.standard.base64
+
+import kotlinx.serialization.KSerializer
+import kotlinx.serialization.descriptors.PrimitiveKind
+import kotlinx.serialization.descriptors.PrimitiveSerialDescriptor
+import kotlinx.serialization.descriptors.SerialDescriptor
+import kotlinx.serialization.encoding.Decoder
+import kotlinx.serialization.encoding.Encoder
+
+object ByteArrayAsBase64StringSerializer : KSerializer<ByteArray> {
```

Comment:
is ks3 also going to include the typealias? It would be more convenient to users.

Reply:
I think we could have a separate package for predefined typealiases. Users who prefer that method of picking serializer can depend on it to use it, and those who prefer another method won't have their autocompletes polluted. Sound ok with you?

Reply:
I think it's okay to include typealiases in one package. It's a bit inconvenient to set up a Maven dependency, and I think the only real problem with autocomplete spam is with extension functions